tests/integration/aliases/aliases_test.go

```diff
@@ -80,3 +80,30 @@ func TestDotNetAliases(t *testing.T) {
 		})
 	}
 }
+
+func TestGoAliases(t *testing.T) {
+	// TODO replace with := range dirs once all passing
+	for _, dir := range []string{
+		"rename",
+		// "adopt_into_component",
+		// "rename_component_and_child",
+		// "retype_component",
+		// "rename_component",
```

Comment:
Will be great to add more of these before merging - as they each do cover independent code paths and feature requirements.

Reply:
Yup, I mentioned this in a TODO in the description. Thanks for taking an early look!